scylla/src/response/rows_displayer.rs

```diff
@@ -0,0 +1,1194 @@
+            f,
+            self.settings.print_in_color,
+            self.settings.colors.uuid,
+            "{}",
+            self.value
+        )
+    }
+}
+
+fn display_in_quotes_and_colored<'a>(
+    f: &mut fmt::Formatter<'_>,
+    value: &(dyn StringConvertible<'a> + 'a),
+    settings: &RowsDisplayerSettings,
+) -> fmt::Result {
+    write_colored!(f, settings.print_in_color, settings.colors.text, "'")?;
+    write_colored!(
+        f,
+        settings.print_in_color,
+        settings.colors.text,
+        "{}",
+        value
+    )?;
+    write_colored!(f, settings.print_in_color, settings.colors.text, "'")
+}
+
+// list or set
+impl fmt::Display for WrapperDisplay<'_, Vec<CqlValue>> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        // choose opening bracket
+        if matches!(self.cql_value, CqlValue::List(_)) {
+            write_colored!(
+                f,
+                self.settings.print_in_color,
+                self.settings.colors.collection,
+                "["
+            )?;
+        } else if matches!(self.cql_value, CqlValue::Set(_)) {
+            // it is a set
+            write_colored!(
+                f,
+                self.settings.print_in_color,
+                self.settings.colors.collection,
+                "{{"
+            )?;
+        } else {
+            panic!("BUG"); // Describe the reason why it might happen.
+        }
+
+        // print elements
+        let mut first_ture = true;
+        let value: &Vec<CqlValue> = self.value;
+        for item in value {
+            let item_wrapper = get_item_wrapper(item, self.settings);
+
+            // write comma
+            if !first_ture {
+                write_colored!(
+                    f,
+                    self.settings.print_in_color,
+                    self.settings.colors.collection,
+                    ", "
+                )?;
+            }
+            first_ture = false;
+
+            // if item is text, display it in colored quotes, otherwise display it as is (e.g. number)
+            // to not confuse with text
+            if matches!(item, CqlValue::Text(_)) {
+                display_in_quotes_and_colored(f, &*item_wrapper, self.settings)?;
+            } else {
+                write!(f, "{}", item_wrapper)?;
+            }
+        }
+
+        // choose closing bracket
+        if matches!(self.cql_value, CqlValue::List(_)) {
+            write_colored!(
+                f,
+                self.settings.print_in_color,
+                self.settings.colors.collection,
+                "]"
+            )
+        } else if matches!(self.cql_value, CqlValue::Set(_)) {
+            // it is a set
+            write_colored!(
+                f,
+                self.settings.print_in_color,
+                self.settings.colors.collection,
+                "}}"
+            )
+        } else {
+            panic!("BUG") // Describe the reason why it might happen.
+        }
```

Comment:
:recycle: If matching has at least 3 branches (and this one indeed has), then it's best to use `match` expression.